scargo/scargo_src/sc_gen.py

```diff
@@ -138,13 +142,17 @@ def gen_fs_esp32(config: Config) -> None:
         sys.exit(1)
 
 
-def gen_single_binary_esp32(project_profile_path: Path, config: Config):
-    partition_list = config.esp32.partitions
+def gen_single_binary_esp32(project_profile_path: Path, config: Config) -> None:
+    logger = get_logger()
+    esp32_config = config.esp32
+    if not esp32_config:
```

Comment:
same

Reply:
ok

Reply:
I misunderstood, so: same, outside the scope

Reply:
Validation added